assets/minecraft/texts/splashes.txt

```diff
@@ -6,9 +6,95 @@ Made by Sake!
 Made by &flower!
 Made by Nexx!
 Made by Skye!
-Made by Laxla!
+Made by Laxystem!
+(We're not a laxlative :kittygun:)
 Made by Felix!
-Made by UnlikePaladin!
+Made by UnlikePaladin!ou
 Made by Ted!
 Made by Echo!
 Check out the Ko-Fi!
+pridecraft.gay
+No cishets were harmed in the making of this pack
+PRIDE
+You are valid.
+You are valid!
+YOU ARE VALID!!!
+people.forEach { it.isValid = true }
+--. .- -.--
+LESBIAN
+GAY
+BI
+TRANS
+QUEER
+ACE
+(The I requested to leave. Respect that!)
```

Comment:
may be confusing to some people, it's mostly still included in the LGBT umbrella afaik

Reply:
afaik the intersex community requested not to be defined by the I